src/goto-instrument/generate_function_bodies.h

```diff
@@ -70,21 +70,36 @@ void generate_function_bodies(
   goto_modelt &model,
   message_handlert &message_handler);
 
+void generate_function_bodies(
+  const std::string &function_name,
+  const std::string &call_site_id,
+  const generate_function_bodiest &generate_function_body,
+  goto_modelt &model,
+  message_handlert &message_handler);
+
 // clang-format off
 #define OPT_REPLACE_FUNCTION_BODY \
   "(generate-function-body):" \
+  "(generate-havocing-body):" \
   "(generate-function-body-options):"
 
 #define HELP_REPLACE_FUNCTION_BODY \
   " --generate-function-body <regex>\n" \
   /* NOLINTNEXTLINE(whitespace/line_length) */ \
   "                              Generate bodies for functions matching regex\n" \
+  " --generate-havocing-body <option>\n" \
```

Comment:
There is a `havoc` in generate function body options, is there a reason why we need a separate option for this?